packages/backend-defaults/report-auditor.api.md

```diff
@@ -0,0 +1,138 @@
+## API Report File for "@backstage/backend-defaults"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import type { AuditorCreateEvent } from '@backstage/backend-plugin-api';
+import type { AuditorEventSeverityLevel } from '@backstage/backend-plugin-api';
+import { AuditorService } from '@backstage/backend-plugin-api';
+import type { AuthService } from '@backstage/backend-plugin-api';
+import type { Config } from '@backstage/config';
+import type { Format } from 'logform';
+import type { HttpAuthService } from '@backstage/backend-plugin-api';
+import type { JsonObject } from '@backstage/types';
+import type { PluginMetadataService } from '@backstage/backend-plugin-api';
+import type { Request as Request_2 } from 'express';
+import { ServiceFactory } from '@backstage/backend-plugin-api';
+import * as winston from 'winston';
+
+// @public
+export class Auditor implements AuditorService {
+  // (undocumented)
+  addRedactions(redactions: Iterable<string>): void;
```

Comment:
Hm. Yes but, it's probably a bad idea to ever audit-log things that can even possibly contain secrets 🤔 

Would have been great with something like https://github.com/backstage/backstage/pull/24730 @Rugvip